src/city_score/sources/tpl_parks.py

```diff
@@ -0,0 +1,60 @@
+from ..cache import cache
+from ..decorators import dimension, scorer
+from ..source import Source
+from html.parser import HTMLParser
+import requests
+
+class TrustForPublicLand(Source):
+    pass
```

Comment:
still need this `class`?

Reply:
I would keep it, because the user instructions say to add all sources to the source tuple, so there should be a class even if it's a stub.